car_control_planners_benchmark:
  ros__parameters:
    planner_timeout: 30.0
    interpolation_parameter: 50
    octomap_voxel_size: 0.2
    selected_state_space: "SE2" # "DUBINS","REEDS", "SE2", "SE3" ### PS. Use DUBINS OR REEDS for Ackermann
    selected_planners: [InformedSGCP]  # RRT,RRTStarF,LQRPlanner,LQRRRTStar,SST,EST, KPIECE1, 
    min_turning_radius: 1.5
    robot_mesh_path: "" # leave empty if you do not have one, robot_mesh_path: ""    
    state_space_boundries:
      minx: -10.0
      maxx: 10.0
      miny: -7.0
      maxy: 7.0
      minz: 0.
      maxz: 0.5
      minyaw: -3.14
      maxyaw: 3.14
      minv: -1.5
      maxv: 1.5
    control_boundries:
      minv: -1.0
      maxv:  1.0
      minw: -1.0
      maxw:  1.0
    robot_body_dimens:
      x: 0.2
      y: 0.2
      z: 0.1
    start:
      z: 1.5
    goal:
      z: 1.5
    goal_tolerance: 1.0
    min_euclidean_dist_start_to_goal: 25.0
    batch_size: 1
    epochs: 100
    max_memory: 4096
    results_output_dir: "/home/atas/test/"
    results_file_regex: "control"
    publish_a_sample_bencmark: true
    sample_bencmark_plans_topic: "benchmark_plan"
    octomap_from_file: "/home/atas/Downloads/quad_world_octomap.bt"

